temboardui/plugins/monitoring/handlers/alerting.py

```diff
@@ -105,6 +105,72 @@ def index(request):
     )
 
 
+@blueprint.instance_route("/alerting/checks.json", methods=['GET', 'POST'])
+def checks(request):
+    host_id, instance_id = get_instance_ids(request)
+    if 'GET' == request.method:
+        data = checks_info(request.db_session, host_id, instance_id)
+        for datum in data:
+            spec = check_specs[datum['name']]
+            if 'value_type' in spec:
+                datum['value_type'] = spec['value_type']
+        return jsonify(data)
+    else:
```

Comment:
What about two different function, one for each method?
This would cut the indentation down a bit.